specify/specify7
# Fix Query Builder Aggregator Display

Fixes #4510

On the production branch, the query `https://sp7demofish2924-edge.test.specifysystems.org/specify/query/87/`
returns
```json
{
  "results": [
      [10924,null,10924,"Francine","Abe",true],
      [10924,null,10924,"Justin","Bagley",false],
      [10924,null,10924,"M","Caire",false]]
}
```
when the collectors aggregator fields should not be null, and should be ordered by `isPrimary` (according to the Collector Aggregator), so in this example `Caire, M | Bagley, Justin | Abe, Francine`

### Checklist

      and self-explanatory (or properly documented)
- [ ] UI User Testing

### Testing instructions
Create queries with aggregators, and make sure there are not any blank aggregator fields, when they shouldn't be.  So, nothing like ` |  |  | `.  Look at the example in the issue.

Also, create queries with aggregators, then compare the aggregator field in the query results vs the record's form.  The ordering should be generally the same for most 'sort by' fields.  When sorting by a Boolean (True/False) fields, like 'IsPrimary', the exact orderings aren't expected to be exactly the same, because of the ambiguity of the ordering of the rows stored in the database.
